office/company_events.md

```diff
@@ -1,7 +1,16 @@
 # source{d} Events
 
 ## Get togethers
-Once a month we organise a get together where we either all go to a bar where free drinks will be served from 19:00-24:00 or we go for some group activities like salsa classes, karting or other fun stuff. Every sourcerer and their +1s are invited to join. Get togethers are the third Wednesday of the month. 
+Once a month we organise a get together where we either all go to a bar where free drinks will be served from 19:00-24:00 or we go for some group activities like salsa classes, karting or other fun stuff. That means, every Get together can be different and always depends on the activity.
+Some activities let us be more flexible (beers) while others have to follow a specific schedule (classes). We can generalize:
+  - The event is on the calendar from now to forever, it is the third Wednesday of the month.
+  - The starting time usually is around 7 pm and the duration is around 2-3 hours. 
+  - The goal of this event is to increase the knowledge within the team, there is not a minimum amount of people required,     but if we see there are not a lot of people joining because of the date (not because of the activity) we will try to       change it to be able to put together more people.
+  - We will tell you if the activity is only for sourcerers or your +1s are also invited to join. 
```

Comment:
I think @estherrgarcia in order to be less work for you and not have for every get together to update people for their +1, I suppose that when it's normal beers the +1 is by default right? And in case we have some different activities then you could update us on that.

Reply:
Yes, beers always include our +1s!